std/json.d

```diff
@@ -1290,7 +1302,13 @@ Set the $(LREF JSONOptions.specialFloatLiterals) flag is set in `options` to enc
 string toJSON(const ref JSONValue root, in bool pretty = false, in JSONOptions options = JSONOptions.none) @safe
 {
     auto json = appender!string();
+    toJSON(json, root, pretty, options);
+    return json.data;
+}
 
+void toJSON(Out)(auto ref Out json, const ref JSONValue root, in bool pretty = false, in JSONOptions options = JSONOptions.none) @safe
```

Comment:
```suggestion
void toJSON(Out)(
    auto ref Out json,
    const ref JSONValue root,
    in bool pretty = false,
    in JSONOptions options = JSONOptions.none) @safe
```

Otherwise it fails the style check with:
```
std/json.d(1309:112)[warn]: Line is longer than 120 characters
```

Reply:
thank you, should be fixed now